GitUI/UserControls/GPGKeys/GPGSecretKeysCombobox.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.ComponentModel;
+using System.Diagnostics;
+using GitCommands.Git.Gpg;
+using GitCommands.Gpg;
+
+namespace GitUI.UserControls.GPGKeys
+{
+    public partial class GPGSecretKeysCombobox : GitModuleControl
```

Comment:
This isn't a combobox control, but a custom control that contains a combobox.
We don't have a consistent naming strategy for custom controls, but based on what we already have I think we should probably name it something like:
```suggestion
    public partial class GpgSecretKeysComboBoxControl : GitModuleControl
```
or
```suggestion
    public partial class GpgSecretKeysSelector : GitModuleControl
```

Reply:
I like selector better.